docs/docsite/rst/userguide/teams.rst

```diff
@@ -118,26 +126,22 @@ To add permissions to a Team:
 3. Click to select the resource to assign team roles and click **Next**.
 
 .. image:: ../common/images/teams-permissions-templates-select.png
+   :alt: Permissions Templates Select
 
 4. Click the checkbox beside the role to assign that role to your chosen type of resource. Different resources have different options available.
 
 .. image:: ../common/images/teams-permissions-template-roles.png
+   :alt: Permissions Template Roles
 
 
 5. Click **Save** when done, and the Add Permissions Wizard closes to display the updated profile for the user with the roles assigned for each selected resource.
```

Comment:
Found a typo in original text - Should be "...updated profile for the team with the roles assigned..."